packages/runtime/src/microapp-runtime.ts

```diff
@@ -23,24 +26,41 @@ export class MicroappRuntime {
     this.#iframe = iframe;
     this.#theme = theme ?? this.#theme;
     this.#lang = lang ?? this.#lang;
+    this.#targetOrigin = new URL(src).origin;
```

Comment:
Do we need it to be a class attr?

```suggestion
    const { origin: targetOrigin } = new URL(src);
```

Reply:
We need it because of the usage when injecting the script at the bottom of the file:

```
function notifyRouteChange(method) {
          const currentRoute = window.location.pathname;
          window.parent.postMessage(
            { 
              type: '@microapp:routeChange', 
              payload: { route: currentRoute }
            }, 
            '${this.#targetOrigin}'
          );
        }
```